### ISSUE TITLE
Inactive kill quest givers

### ISSUE THREAD

**Issue by Talissera** [WarEmu/WarBugs#1589]
Hello.
Have a problem with repeatable quests on destro side (To kill 15/25 players) in Ostland and Barak Varr/Marsh of Madness. Quest NPCs have blue map tag and inactive when I click on it.

On Order side there is no such problem.

**Comment by Reggaewaldo** [comment]
Can you please give me the name of the quest when you have the time and I will gladly have a look at it. I suspect there is an easy fix for it.

**Comment by Evildoor** [comment]
My guess will be "Stoking the Fire" and "Raven God's Tithe" in Ostland (pardon my possible mistakes, i can't access the game and check the exact quest names right now), can't remember BV/MoM quest names. Some additional info: it seems to me that somehow a level requirement was added to these quests - i have no problems with them on my lvl 12-26 chars, but can't take them as lvl 8 BO. On order side similar quests can be done by everyone, including lvl 8 AM.

**Comment by Talissera** [comment]
If you have done smth - Thank you a lot, now it works!

**Comment by Reggaewaldo** [comment]
I have not done anything but I'm guessing that you are correct in your assumtion Evil. I will have a look once the Dev server is avail again.

**Comment by Talissera** [comment]
26 lvl. Same problem with BV NPC Grub Broke-Nose. Sometimes I can finish quest (didn't start it), and then he instantly has blue symbol.

**Comment by Evildoor** [comment]
Update: according to more reports my assumption was wrong, and quest givers just sometimes refuse to give quests to some characters of any level.

**Comment by Abolition** [comment]
I opened a ticket for this, but it got closed with a redirect to this thread, so I just wanted to post what I am experiencing, in case it is of any help in squashing the bug.

Yesterday there were a couple of server restarts. After the first restart, but before the second restart, I turned in almost all of my completed kill quests (did the round-the-world trip). However, I could not start the quests again.

I figured this was an issue that would be resolved after a restart, however this is not the case. I simply cannot see icons above the heads of the NPCs and right clicking on them does nothing.

However, there is one kill quest that I had not started at the time (keep lord in Barak Varr). In the days since the server restarts I am able to start, complete and restart this one quest with no problems. Only the quests that I turned in during the "buggy phase" are still not able to be picked up, which is every other kill quest.

As far as I can tell - this is isolated to my Bright Wizard only, as this was the only character I was playing during this server restart period. My other characters, on both realms, are able to complete kill quests as normal.

It appears I am not the only one - I made a thread in the Game Support forums and a couple other people reported the same issue. I have also spoken to numerous people in game with the same problem. The same issue being that it is directly related to these specific server restarts.

Perhaps the server logged us as completing a quest that was no longer repeatable?

Feel free to contact me in game.

BW: AboMeltsYou

Destro: Abolition, Abogroupheals
